tests/unit_tests/test_load_forward_model.py

```diff
@@ -69,48 +69,6 @@ def run_simulator(time_step_count, start_date) -> Summary:
     return summary
 
 
-@pytest.mark.usefixtures("copy_snake_oil_case_storage")
-def test_load_inconsistent_time_map_summary(caplog):
```

Comment:
Is this covered by the new tests? Seems a bit unrelated?

Reply:
It only checked that you got a warning that we don't expect to get any more.